### PR TITLE
Translatons: Initial translation to ka/georgian language

### PR DESCRIPTION
**Pull request by GioMac**

### CODE REVIEW

**Comment by rousskov** [comment]
OK to test

**Comment by rousskov** [comment]
Thanks a lot for submitting these translations! I am not qualified to review them, unfortunately, but I adjusted the PR title to better match recent translation-related commits. The PR is in @yadij hands now.

File: errors/TRANSLATORS [hunk 1]
```diff
@@ -48,6 +48,7 @@ via Rosetta Translation Project and Canonical Ltd.
 	Fly-Man- 
 	Fredrik Svensson <fredrik@member.fsf.org>
 	FrenzY <frenzy.ivan@gmail.com>
+	George Machitidze <giomac@gmail.com>
```
[the remark this answers is not in the document]
Comment (GioMac) [reply errors/TRANSLATORS]: Not from Rosetta - there is no single mention about rosetta or how translations should be made:
1. Translations site is not working
2. Even google cannot point to Rosetta of you search for "squid proxy translation"

[the remark this answers is not in the document]
Comment (yadij) [reply errors/TRANSLATORS]: Okay. Then please update this PR to add your name to the first list of names in TRANSLATORS file.

**Comment by rousskov** [comment]
retest please

@eduard-bagdasaryan, when you get a chance, please check why this PR is ignored by Jenkins and/or reset/start those Jenkins tests if needed.

**Pull request closed without merging**